modules/transports/core/nhttp/src/main/java/org/apache/synapse/transport/passthru/PassThroughHttpSender.java

```diff
@@ -268,6 +268,8 @@ public InvocationResponse invoke(org.apache.axis2.context.MessageContext msgCont
                         && msgContext.getProperty(PassThroughConstants.ORGINAL_CONTEN_LENGTH) != null) {
                     msgContext.setProperty(PassThroughConstants.PASSTROUGH_MESSAGE_LENGTH, Long.parseLong(
                             (String) msgContext.getProperty(PassThroughConstants.ORGINAL_CONTEN_LENGTH)));
+                } else if (msgContext.getProperty(PassThroughConstants.DISABLE_CHUNKING) == null) {
+                    msgContext.setProperty(PassThroughConstants.DISABLE_CHUNKING_CHECK, false);
```

Comment:
It seems we always set this property to false. Do we need to introduce a new property for `DISABLE_CHUNKING_CHECK`?